sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -260,9 +260,6 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession with SQLHelper
       newLine.startsWith("--") && !newLine.startsWith("--QUERY-DELIMITER")
     }
 
-    // SPARK-32106 Since we add SQL test 'transform.sql' will use `cat` command,
-    // here we need to check command available
-    assume(TestUtils.testCommandAvailable("/bin/bash"))
```

Comment:
Does it fail or skip? I think it skips, which is fine?

Reply:
Seems it skips all tests

Reply:
Can we conditionally skip here instead of filtering the tests out?

Reply:
If fail here, `SQLQueryTestSuite` exit.